windfile.py

```diff
@@ -0,0 +1,111 @@
+import numpy as np
+import math
+import elev
+from datetime import datetime, timedelta, timezone
+import bisect
+import time
+
+class WindFile:
+    def __init__(self, path: str):
+        self.data_npz = np.load("downtest2/" + path, "r")
+        self.data = self.data_npz['data']
+        self.time = self.data_npz['timestamp']
+        self.altitude = self.data_npz['levels']
+        self.interval = self.data_npz['interval']//3600
+        # TODO calculate resolution
+
+    def get(self, lat: float, lon: float, altitude: float, time: float):
+        if lat < -90 or lat > 90 or lon < -180 or lon > 360 or altitude > 975: # TODO cannot know altitude
+            raise Exception("Invalid input value") # TODO more precise (which axis)
+
+        # TODO wrap longitude
+
+        bounds = self.get_bounds_and_fractions(lat, lon, altitude, time)
+        
+        #diffs = GEFS_ALT_DIFFS if levels == GEFS else GFSHIST_ALT_DIFFS # ???? not sure if this is still needed
+        
+        u, v = self.get_wind_helper(*bounds)
+        
+        return u, v
+
+    def get_bounds_and_fractions(self, lat: float, lon: float, alt: float, time: float):
+        lat_res, lon_res, pressure_res = None, None, None
+        
+        lat = 90 - lat
+        lon = lon % 360
+
+        base_time = int(datetime.fromtimestamp(self.time).strftime("%Y%m%d%H"))
+        time = (time - base_time)/self.interval
```

Comment:
For this to work self.interval needs to be in seconds. Also, clients will likely want to pass in `time` arguments that are `datetime` objects, so you should check the type and call `timestamp` if necessary